components/trezor_bridge/resources/app.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import { render } from 'react-dom'
+import { sendWithPromise } from 'chrome://resources/js/cr.m'
+
+// Hooks to trick some checks inside TrezorConnect to use webextension env.
+function setupBraveHooks () {
+  if (!window.chrome) {
+    window.chrome = Object({})
+  }
+  if (!window.chrome.runtime) {
+    window.chrome.runtime = Object({})
+  }
+  if (!window.chrome.runtime.id) {
+    window.chrome.runtime.id = ''
+  }
+  if (!window.chrome.runtime.onConnect) {
+    window.chrome.runtime.onConnect = Object({
+      addListener: () => undefined
+    })
+  }
```

Comment:
hmm, I'm not sure why we would need to stub these APIs if we're using the browser version of trezor-connect and not the webextension version...

Reply:
we are using the webextension version, it doesnt have `browser` version: `"node" | "web" | "webextension" | "electron" | "react-native" | undefined`

Reply:
changed to `web`